ui/webui/src/app.jsx

```diff
@@ -33,15 +41,60 @@ export const Application = () => {
 
     useEffect(() => cockpit.file("/run/anaconda/bus.address").watch(setAddress), []);
 
+    const steps = [
+        {
+            id: 0,
+            name: "Installation language",
+            component: <AddressContext.Provider value={address}>
+                <InstallationLanguage />
+            </AddressContext.Provider>
+        },
+        {
+            id: 1,
```

Comment:
Where did you copy this from? Since that the id property here is not consumed by Patternfly.

Reply:
Following up to our pm discussion https://github.com/patternfly/patternfly-react/issues/6993

Reply:
I took the numeric ids from: https://www.patternfly.org/v4/components/wizard/react-demos#in-modal

Reply:
Removed.